docs/gke/configs/cluster.jinja

```diff
@@ -108,7 +108,12 @@ resources:
           nodeMetadata: SECURE
         {% endif %}
         oauthScopes:
-        - gke-default
+        - https://www.googleapis.com/auth/devstorage.read_only
+        - https://www.googleapis.com/auth/logging.write
+        - https://www.googleapis.com/auth/monitoring
+        - https://www.googleapis.com/auth/service.management.readonly
+        - https://www.googleapis.com/auth/servicecontrol
```

Comment:
logging and monitoring make sense for the logging and monitoring agents but what about the others?

I assume devstorage is for GCR?

Why do we need service.management, servicecontrol, and trace?

Reply:
I substituted `gke-default` [1] with the actual scopes. I will reduce the scopes.

[1] https://cloud.google.com/sdk/gcloud/reference/container/clusters/create